src/BulletCollision/CollisionDispatch/btCollisionObject.h

```diff
@@ -25,8 +25,8 @@ subject to the following restrictions:
 #define DISABLE_DEACTIVATION 4
 #define DISABLE_SIMULATION 5
 
-struct	btBroadphaseProxy;
```

Comment:
Looks like tabs in this file got automatically converted to spaces. Please undo so that this file is not part of the change set.